### ISSUE TITLE
Crash when tapping on specific conversations in the conversation list on Android

### ISSUE THREAD

**Issue by chris-armstrong** [intercom/intercom-cordova#126]
## Version info
  - Tested on 3.0.20, 3.0.19, 3.0.18 and git master
  - Android OS version: 6.0.1
  - Android Device: Samsung S7, Sony Experia X

## Actual behavior

Tapping on certain conversations in the conversation list opens the conversation window with a spinner, but the entire app crashes soon after. The issue is reliably reproduceable.

## Steps to reproduce
 1. Display the conversation list
 2. Tap on a problematic conversation => opens conversation window with spinner
=> app crash

## Logs

Here is a stack trace from the crash:

```
12-14 10:24:04.877 25384-25384/com.podtrackers.app E/AndroidRuntime: FATAL EXCEPTION: main
   Process: com.podtrackers.app, PID: 25384
   java.lang.RuntimeException: Could not dispatch event: class io.intercom.android.sdk.models.events.ConversationEvent to handler [EventHandler public void io.intercom.android.sdk.conversation.ConversationFragment.conversationSuccess(io.intercom.android.sdk.models.events.ConversationEvent)]: Attempt to invoke interface method 'android.view.View io.intercom.android.sdk.blocks.blockInterfaces.NotificationChannelsCardBlock.addNotificationChannelsCardBlock(io.intercom.android.sdk.blocks.models.NotificationChannelsCard, boolean, boolean, android.view.ViewGroup)' on a null object reference
       at io.intercom.com.squareup.otto.Bus.throwRuntimeException(Bus.java:460)
       at io.intercom.com.squareup.otto.Bus.dispatch(Bus.java:387)
       at io.intercom.com.squareup.otto.Bus.dispatchQueuedEvents(Bus.java:368)
       at io.intercom.com.squareup.otto.Bus.post(Bus.java:337)
       at io.intercom.android.sdk.MainThreadBus.post(MainThreadBus.java:18)
       at io.intercom.android.sdk.api.CallbackHolder$6.onSuccess(CallbackHolder.java:116)
       at io.intercom.android.sdk.api.CallbackHolder$6.onSuccess(CallbackHolder.java:112)
       at io.intercom.android.sdk.api.BaseCallback.onResponse(BaseCallback.java:31)
       at io.intercom.retrofit2.ExecutorCallAdapterFactory$ExecutorCallbackCall$1$1.run(ExecutorCallAdapterFactory.java:68)
       at android.os.Handler.handleCallback(Handler.java:739)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:158)
       at android.app.ActivityThread.main(ActivityThread.java:7229)
       at java.lang.reflect.Method.invoke(Native Method)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)
    Caused by: java.lang.NullPointerException: Attempt to invoke interface method 'android.view.View io.intercom.android.sdk.blocks.blockInterfaces.NotificationChannelsCardBlock.addNotificationChannelsCardBlock(io.intercom.android.sdk.blocks.models.NotificationChannelsCard, boolean, boolean, android.view.ViewGroup)' on a null object reference
       at io.intercom.android.sdk.blocks.BlockType$6.generateView(BlockType.java:45)
       at io.intercom.android.sdk.blocks.Blocks.createBlocks(Blocks.java:29)
       at io.intercom.android.sdk.conversation.ConversationFragment.createMessageUi(ConversationFragment.java:1227)
       at io.intercom.android.sdk.conversation.ConversationFragment.createConversationUi(ConversationFragment.java:827)
       at io.intercom.android.sdk.conversation.ConversationFragment.setupWithConversation(ConversationFragment.java:818)
       at io.intercom.android.sdk.conversation.ConversationFragment.conversationSuccess(ConversationFragment.java:736)
       at java.lang.reflect.Method.invoke(Native Method)
       at io.intercom.com.squareup.otto.EventHandler.handleEvent(EventHandler.java:89)
       at io.intercom.com.squareup.otto.Bus.dispatch(Bus.java:385)
       at io.intercom.com.squareup.otto.Bus.dispatchQueuedEvents(Bus.java:368) 
       at io.intercom.com.squareup.otto.Bus.post(Bus.java:337) 
       at io.intercom.android.sdk.MainThreadBus.post(MainThreadBus.java:18) 
       at io.intercom.android.sdk.api.CallbackHolder$6.onSuccess(CallbackHolder.java:116) 
       at io.intercom.android.sdk.api.CallbackHolder$6.onSuccess(CallbackHolder.java:112) 
       at io.intercom.android.sdk.api.BaseCallback.onResponse(BaseCallback.java:31) 
       at io.intercom.retrofit2.ExecutorCallAdapterFactory$ExecutorCallbackCall$1$1.run(ExecutorCallAdapterFactory.java:68) 
       at android.os.Handler.handleCallback(Handler.java:739) 
       at android.os.Handler.dispatchMessage(Handler.java:95) 
       at android.os.Looper.loop(Looper.java:158) 
       at android.app.ActivityThread.main(ActivityThread.java:7229) 
       at java.lang.reflect.Method.invoke(Native Method) 
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230) 
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120) 
```

**Comment by ronocod** [comment]
This is fixed in the latest Android SDK release, I'll update the Cordova SDK today to use that version.

**Comment by ronocod** [comment]
This is fixed in 3.0.21.
